docs/contributing/committers.md

```diff
@@ -85,8 +85,8 @@ Summary:
 
 If they accept, then do:
 
-1. Add to Committer team of GitHub Islandora organization ??
```

Comment:
With the split in Islandora 7 and 8, should this be 
> Add to the Islandora 8 Committer team of the Github Islandora organization.

Reply:
Right-o!